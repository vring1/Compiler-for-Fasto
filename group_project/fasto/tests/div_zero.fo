fun int main() =
    let a = 10 in
    let b = 0  in
    let c = a/b in
    write(c)