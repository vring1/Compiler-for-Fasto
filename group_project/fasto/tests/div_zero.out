Runtime error at line 0: division by zero
